packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -93,8 +93,8 @@ const Accordion = ({
         {...panelElement.props}
       >
         <PanelTrigger
-          iconCollapsed="triangle-right"
-          iconExpanded="triangle-down"
+          iconCollapsed="triangle-solid-right"
+          iconExpanded="triangle-solid-down"
```

Comment:
Need to check with @dequelabs/cauldron-design on this, I'm not sure we need 2 different styles of triangle icons. It's possible that it's intended for this triangle to replace the existing one.

Reply:
For now, keep the existing icon. There will be a bunch of work in the future to address the greater body of work where the icons don't match up with the design system (https://github.com/dequelabs/cauldron/issues/869)